src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -101,6 +82,22 @@ void createStationWithDuplicateName() {
     @DisplayName("지하철역을 조회한다.")
     @Test
     void getStations() {
+        // given
+        String stationName = "강남역";
+        String stationName2 = "역삼역";
+        // when
+        지하철역_생성(stationName);
+        지하철역_생성(stationName2);
+
+        // then
+        List<String> stationNames = 지하철역_조회();
+
+        // then
+        assertAll(
+                () -> assertThat(stationNames.size()).isEqualTo(2),
+                () -> assertThat(stationNames).containsAnyOf("강남역"),
+                () -> assertThat(stationNames).containsAnyOf("역삼역")
+        );
```

Comment:
이 부분 역시 필수는 아니지만 다음과 같이 검증을 작성하실 수도 있으니 참고만 부탁드리겠습니다. 🙇‍♂️
```suggestion
        assertThat(stationNames)
                .hasSize(2)
                .containsExactlyInAnyOrder("강남역", "역삼역");
```